ports/libsmb2/vcpkg.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "libsmb2",
+  "version-date": "2021-04-29",
+  "description": "SMB2/3 Userspace Client",
+  "homepage": "https://github.com/sahlberg/libsmb2",
+  "supports": "!osx",
+  "dependencies": [
+    "openssl",
+    "vcpkg-cmake-config"
```

```suggestion
    {
      "name": "vcpkg-cmake-config",
      "host": true
    }
```

`vcpkg-cmake-config` is a helper script and so should only be installed for the host.